README.md

```diff
@@ -112,17 +112,16 @@ You may need to add manual [references](http://www.typescriptlang.org/docs/handb
 
 Definitely Typed only works because of contributions by users like you!
 
-### Test
+### Testing
 
 Before you share your improvement with the world, use it yourself.
 
 #### Test editing an existing package
 
-To add new features you can use [module augmentation](http://www.typescriptlang.org/docs/handbook/declaration-merging.html#module-augmentation).
-You can also directly edit the types in `node_modules/@types/foo/index.d.ts`, or copy them from there and follow the steps below.
+To test local to your project, you can use [module augmentation](http://www.typescriptlang.org/docs/handbook/declaration-merging.html#module-augmentation) to extend existing types.
```

Comment:
It's not super obvious that we're talking about (1) your own project that uses (2) the package you want to improve. I'm not sure how best to distinguish those two things, though.